cleanlab/datalab/internal/issue_manager/data_valuation.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (C) 2017-2023  Cleanlab Inc.
+# This file is part of cleanlab.
+#
+# cleanlab is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cleanlab is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with cleanlab.  If not, see <https://www.gnu.org/licenses/>.
+from __future__ import annotations
+
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    ClassVar,
+    Dict,
+    List,
+    Optional,
+    Union,
+)
+
+import numpy as np
+import pandas as pd
+from scipy.sparse import csr_matrix
+
+from cleanlab.datalab.internal.issue_manager import IssueManager
+
+if TYPE_CHECKING:  # pragma: no cover
+    import pandas as pd
+    from cleanlab.datalab.datalab import Datalab
+
+
+class DataValuationIssueManager(IssueManager):
+    """Manages data sample with low valuation."""
+
+    description: ClassVar[
+        str
+    ] = """Examples who won't contribute a lot for a model's training
+     are flagged as having label issues.
```

Comment:
Wait, should we compare this issue with label issues?

Reply:
sorry this is a mistake